[DOCS-1956] Update docs for hashed API keys (#2018)
wandb/docs: snippets/en/_includes/api-key-view-once-warning.mdx

<Warning>

> The full API key is only shown once at creation time. After you close the dialog, you will not be able to view the full API key again. Only the key ID (first part of the key) will be visible in your settings. If you lose the full API key, you must create a new API key.

Reviewer: ```suggestion
The full API key is only shown once at creation time. After you close the dialog, you cannot view the full API key again. Only the key ID (first part of the key) is visible in your settings. If you lose the full API key, you must create a new API key.
```
This is future tense that I think we can easily eliminate.
